Woof/Woof/Model/Owner/Owner+CurrentOwner.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+extension Owner {
+    /**
+     Handles the operations with current owner.
+     */
+    struct CurrentOwner {
+        // MARK: - Public Interface
+
+        init(_ storageName: String) {
```

Comment:
Could you please explain why the current owner is parametrised with the storage name?

Reply:
My point is to add it for testing propose only. It allows us to create separate test storage.